server/src/main/java/org/spine3/server/aggregate/Apply.java

```diff
@@ -26,7 +26,8 @@
 import java.lang.annotation.Target;
 
 /**
- * Marks a method of an aggregate as one that modifies the state of the aggregate with data from the passed event.
+ * Marks a method of an aggregate as one that modifies the state of the aggregate with data
+ * from the passed event.
```

Comment:
@alexander-aleksandrov Yes, I am proposing  to make the width for each line more or less equal, as you did it above.